src/microns_to_nwb/tools/ophys/ophys.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+from hdmf.backends.hdf5 import H5DataIO
+from phase3 import nda, func
+from pynwb.image import GrayscaleImage
+from pynwb.ophys import (
+    RoiResponseSeries,
+    Fluorescence,
+    ImageSegmentation,
+    OpticalChannel,
+)
+
+from tools.nwb_helpers import check_module
+
+
+def add_summary_images(field_key, nwb):
+    ophys = nwb.create_processing_module("ophys", "processed 2p data")
+
+    correlation, average = (nda.SummaryImages & field_key).fetch1("correlation", "average")
+
+    correlation_image = GrayscaleImage(correlation)
+    average_image = GrayscaleImage(average)
+
+    ophys.add(correlation_image)
+    ophys.add(average_image)
+
+
+def add_plane_segmentation(field_key, nwb, imaging_plane, image_segmentation):
+    ps = image_segmentation.create_plane_segmentation(
```

Comment:
```suggestion
    plane_segmentation = image_segmentation.create_plane_segmentation(
```

For maximum readability, I do recommend using more unique variable names, including the full name of what the variable represents